src/components/MovieDetails/MovieDetails.css

```diff
@@ -0,0 +1,128 @@
+.movie-details {
+	height: 100%;
+	width: 100%;
+	background: #232323;
+	box-sizing: border-box;
+}
+
+.movie-details__nav {
```

Comment:
for such css class namings better to use scss. .movie {
  &-details{
  & __nav {}
}
}

Reply:
Ack, as I can see in Home task (Task 11) I should implement different approach to styling - maybe this will be a right moment to set up sass, wdyt?

Reply:
Ok